scripts/database.js

```diff
@@ -506,4 +506,11 @@ export class Database {
 
         return this.#instances.get(name);
     }
+
+    static async removeInstance() {
+        const name = cChainParams.current.name;
+        const instance = this.#instances.get(name);
+        if (instance) await instance.removeAccount({ publicKey: null });
+        this.#instances.delete(name);
+    }
```

Comment:
This doesn't actually remove the instance. Instead `Database::removeAccount` should be called

Reply:
nice catch